src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -0,0 +1,89 @@
+import { Icon, IconButton, makeStyles, Paper } from "@material-ui/core";
+import React from "react";
+
+type iconPosition = "left" | "right";
+type arrowStyle = "upDown" | "rightLeft";
+
+export interface SectionProps {
+    isOpen?: boolean;
+    setOpen?: (open: boolean) => void;
+    children: React.ReactNode;
+    title: React.ReactNode;
+    collapsible?: boolean;
+    elevation?: number;
+    iconPos?: iconPosition;
+    classProps?: {
+        section?: string;
+        header?: string;
+        content?: string;
+    };
+    arrowStyle?: arrowStyle;
+}
+
+export const Section = ({
```

Comment:
I see you preferred to define children explicitly in the props instead of using React.FC<Props>. We typically use FC, BUT since it adds always children, we lose some control, so let's go with this. Eventually, we can decide on creating our own "FC", but no need to to it here.